Stripe/PublicHeaders/STPKlarnaLineItem.h

```diff
@@ -0,0 +1,73 @@
+//
+//  STPKlarnaLineItem.h
+//  Stripe
+//
+//  Created by David Estes on 11/19/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ The type of the Klarna line item.
+*/
+typedef NS_ENUM(NSUInteger, STPKlarnaLineItemType) {
+    /**
+     The line item for a product
+     */
+    STPKlarnaLineItemTypeSku,
+    /**
+     The line item for taxes
+     */
+    STPKlarnaLineItemTypeTax,
+    /**
+     The line item for shipping costs
+     */
+    STPKlarnaLineItemTypeShipping
+};
+
+/**
+ An object representing a line item in a Klarna source.
+ @see https://stripe.com/docs/sources/klarna#create-source
+ */
+
+@interface STPKlarnaLineItem : NSObject
+
+/**
+ The line item's type. One of `sku` (for a product), `tax` (for taxes), or `shipping` (for shipping costs).
+ */
+@property (nonatomic) STPKlarnaLineItemType itemType;
+
+/**
+ The human-readable description for the line item.
+ */
+@property (nonatomic, copy) NSString *itemDescription;
+
+/**
+ The quantity to display for this line item.
+ */
+@property (nonatomic, copy) NSNumber *quantity;
+
+/**
+ The total price of this line item.
+ Note: This is the total price after multiplying by the quantity, not
+ the price of an individual item.
+ */
+@property (nonatomic, copy) NSNumber *totalAmount;
```

Comment:
surprised that this isn't paired with a `currency` field. Is that intentional?
EDIT: currency is top-level on the source params. Maybe just add a comment saying that?

Reply:
It exists in the underlying API, but I was trying to reduce boilerplate by making individual objects not have a currency, as it isn't valid for the individual items to have a different currency from the Source. If you found it surprising, maybe I should re-add currency here?

Reply:
hmm yeah I'd probably lean towards adding it, I guess just with the comment that all the currencies need to match...Maybe we can say if it's left null it will default to the currency of the source?

Reply:
@yuki-stripe @mshafrir-stripe thoughts? 
1) It _feels_ weird to set a Money Amount without specifying currency to me
2) But having a separate currency param on each line item feels like an easy way to make mistakes or at the very least write a bunch of redundant code
3) But the API has the currency param on each

Solutions:
1) Leave as is (no currency param)
2) Optional currency param (default to currency of owning Source)
3) "Factory" line items generator? This isn't really typical in objective-c, but I'm trying to think through true use cases for this. Like maybe you can't instantiate a line item separately, only through `SourceParams.addKlarnaLineItem(type, description, ...)`?

Reply:
I suspect users won't be as surprised at the lack of currency.  I think I prefer not exposing it (1) with a note in the docstring ("Note: The amount is denominated in the currency of the top-level STPSourceParams")

Reply:
I am in favor of not including `currency` for the guardrails benefit. Assuming that this class will only be used in the context of a `STPSourceParams`, I don't think it will result in confusion. A docstring will help as well as @yuki-stripe mentioned.

Is `STPKlarnaLineItem` used for both creation (i.e. params) and for the response object, or only creation?